pyfr/plugins/writer.py

```diff
@@ -22,6 +28,9 @@ def __init__(self, intg, cfgsect, suffix=None):
         # Output field names
         self.fields = intg.system.elementscls.convarmap[self.ndims]
 
+        # Output data type
+        self.fpdtype = intg.backend.fpdtype
```

Comment:
Wouldn't it be interesting to give the users to output solution files in other than the default precision? (opt-in feature of course)

Reply:
We can certainly add that without too much hassle.